ui/app/templates/components/transit-key-action/verify.hbs

```diff
@@ -1,4 +1,4 @@
-<form {{action "doSubmit" (hash input=input signature=signature signature_algorithm=signature_algorithm hmac=hmac hash_algorithm=hash_algorithm context=context prehashed=prehashed) on="submit"}}>
+<form {{action "doSubmit" (hash input=input signature=signature signature_algorithm=signature_algorithm hmac=hmac hash_algorithm=hash_algorithm context=context prehashed=prehashed encodedBase64=encodedBase64) on="submit"}}>
```

Comment:
this is a total nit, but would you mind adding linebreaks and indentation here? it'll be a lot easier to read 👀